lib/updates.ml

```diff
@@ -4,9 +4,6 @@
 open Stdext
```

Comment:
This is using stdext, although stdext is not listed as a dependency (it probably comes as a transitive one). Can you please remove all the `open Stdext` and replace the use of the library with the minimal subset of new sublibraries? In this case it would be `Xapi_stdext_pervasives.Pervasiveext` and I believe replacing it with `let finally = Xapi_stdext_pervasives.Pervasiveext.finally` should be enough in this case

Reply:
And there is a reference to `Xapi_stdext_threads.Threadext` half way through.

Reply:
Pushed, I also needed `Xapi_stdext_monadic`. The `finally` was not needed, when I added the open for `Pervasiveext`.